test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -939,6 +944,10 @@ class test_ucp_sockaddr : public ucp_test {
 
         listen_and_communicate(false, SEND_DIRECTION_BIDI);
 
+        if (!has_rndv_lanes(sender())) {
+            UCS_TEST_SKIP_R("no RNDV lanes");
+        }
```

Comment:
why need this check now - how did it pass before?

Reply:
Before that it worked through AM-based RNDV. This patch fixes `RNDV_SCHEME` but the drawback is that when both `RNDV_SCHEME` and `RNDV_THRESH` are set and `RNDV_SCHEME` protocol is not available, `RNDV_THRESH` is not considered anymore. 

So if in that case there is no RNDV lanes, eager can be chosen.

Reply:
When RNDV_THRESH is not set and RNDV_SCHEME is not available, i would expect a fallback to AM-based rndv, since RDNV_THRESH should have higher "priority".

Reply:
So if the only setting is "RNDV_SCHEME=get_zcopy" and this protocol is not available then falling back to AM-based RNDV is preferrable over eager protocols?

Reply:
i guees @yosefe meant when RNDV_THRESH is set, like it is in this case

Reply:
> i guees @yosefe meant when RNDV_THRESH is set, like it is in this case

Right, i've meant set